### PR TITLE
Set up some navigation layout

### PR DESCRIPTION
**Pull request by alex-page**
Uhhh this is a mess. I want to start grouping components together:

Navigation
- Navigation Header
- Navigation List
- Navigation Icon

I might give that a go before merging this ^

Also a bit confused about style scopes. There are some global.. but should I just be making 100 components with scoped styles? e.g. `hr` could be a component, `list` could be a component...  not sure 🤔

### CODE REVIEW

File: src/Navigation.js [hunk 1]
```diff
@@ -1,42 +1,91 @@
...
+			textDecoration: 'none',
+		},
+	};
+
+	const listStyle = {
+		listStyle: 'none',
+		display: 'block',
+		padding: '0 5rem 0 0',
+		margin: 0,
+	};
```
Comment (dominikwilkowski) [inline src/Navigation.js:42]: As soon as you encounter things you want to reuse, I would make a new component. I would not separate styles from the component (as we used to do back in the days) and keep things that work together, together.
Only separate concerns, not technologies.

So yeah we should have a primitive for lists.

In this case `Link` should have that style backed in, so you wrap `Link` and add the style and then just use that component below.

**Comment by dominikwilkowski** [comment]
We should only use global styles where global styles are needed (e.g. resets and theming)
Everything else is co-located with their own concerns.
We also need to built out our primitive components. It should contain common components like buttons, lists, links etc

**Comment by alex-page** [comment]
@dominikwilkowski I agree but do you have an opinion on folder structure? I haven't used emotion before 😬

**Comment by dominikwilkowski** [comment]
I already have a primitive folder in there. We can add a monorepo in there if you like so we don't have to `import X from '.../.../.../.../.../.../.../......../etc';` but use symlinks. Any of that is possible. I didn't think we need it but happy to add it.

**Comment by alex-page** [comment]
I will start using `primitive`.

**Pull request closed without merging**